examples/SIR-direct.jl

```diff
@@ -0,0 +1,60 @@
+using Random
+using Plots
+using Distributions
+using Fleck
+
```

Comment:
This wants a documentation comment at the top to say what it's an example of.

Reply:
Long term what we can do is use Literate.jl and combine with Documenter.jl to build website docs from this. Or if someone write a Julia interface to Quarto that lets us directly build docs websites using Quarto that would be preferable. What [AlgebraicPetri.jl](https://github.com/AlgebraicJulia/AlgebraicPetri.jl) does is take the specially formatted .jl files in examples/ and then uses Literate.markdown to generate markdown docs in docs/build that Documenter.jl then weaves into the final docs webpage.

Reply:
I haven't seen Literate.jl. I'm guessing it follows the ancient "literate programming" scheme. I'll take a look.

Reply:
I don't think we need to look now, and I can set it up when the time comes.